portfolio/src/main/java/com/google/sps/servlets/LoginStatusServlet.java

```diff
@@ -28,9 +28,20 @@
  */
 @WebServlet("/login-status")
 public class LoginStatusServlet extends HttpServlet {
+  private static UserService userService;
+  private static String loginUrl;
+  private static String logoutUrl;
+
+  @Override
+  public void init() {
+    userService = UserServiceFactory.getUserService();
+    logoutUrl = userService.createLogoutURL("/");
+    loginUrl = userService.createLoginURL("/");
+  }
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    UserService userService = UserServiceFactory.getUserService();
+    response.setContentType("text/html");
 
     if (userService.isUserLoggedIn()) {
```

Comment:
Are you planning to do similar checks in the code that actually handles deleting comments etc?

Reply:
Yes, I'll add it as a TODO.